src/main/resources/resources/logisim/strings/std/std.properties

```diff
@@ -819,6 +819,7 @@ TTL747266 = 747266: quad 2-input XNOR gate
 TTL7474 = 7474: dual D-Flipflops with preset and clear
 TTL7485 = 7485: 4-bit magnitude comparator
 TTL7486 = 7486: quad 2-input XOR gate
+TTL7487 = 7487: 4-bit True/complement, zero/one
```

Comment:
Please add commented-out placeholder for this localisation string to all other supported langues.

Reply:
Added in recent commit

Reply:
The component name is incomplete according to the [data sheet](https://archive.org/details/bitsavers_tidataBookVol2_45945352/page/n403/mode/2up). Please, use "4-bit True/complement, zero/one elements"

Reply:
The string is still incomplete, as "elements" is missing.

Reply:
Sorry about that it should be fixed now